docs/compatibility.txt

```diff
@@ -0,0 +1,59 @@
+=============
+Compatibility
+=============
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+JDBC
+====
+
+This JDBC driver follows the JDBC 4.1 standard as it uses PGSQL driver as the
+underlying connector, however there are certain features that are not fully
+compatible with CrateDB.
+
+**The following features are not supported:**
+
+* ``java.sql.ParameterMetaData`` as returned by e.g. ``java.sql.PreparedStatement``
+* ``DataSource``
+* ``CallableStatement`` as CrateDB does not support stored procedures
+
+For further details about compatibility with all possible JDBC features,
+see the ``ResultSet``, ``ResultSetMetaData`` and ``DatabaseMetaData``
```

Comment:
links to the implementations

Reply:
can you provide the links? wasn't able to find what I am supposed to link to

Reply:
hm I don't know what andrei means here, let's wait for him...

Reply:
Probably: https://github.com/crate/pgjdbc/blob/77a2d396dfe8ec89081894f8d0150a85129c6bd0/pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java
https://github.com/crate/pgjdbc/blob/77a2d396dfe8ec89081894f8d0150a85129c6bd0/pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSetMetaData.java and https://github.com/crate/pgjdbc/blob/77a2d396dfe8ec89081894f8d0150a85129c6bd0/pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

Reply:
I was thinking of links to these interfaces we mention: 
- https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html
- https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSetMetaData.html
- https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html